src/cmd/linuxkit/run_vcenter.go

```diff
@@ -43,21 +43,21 @@ func runVcenter(args []string) {
 	flags := flag.NewFlagSet("vCenter", flag.ExitOnError)
 	invoked := filepath.Base(os.Args[0])
 
-	newVM.vCenterURL = flags.String("url", os.Getenv("VCURL"), "URL in the format of https://username:password@host/sdk")
-	newVM.dsName = flags.String("datastore", os.Getenv("VMDATASTORE"), "The Name of the DataStore to host the VM")
-	newVM.networkName = flags.String("network", os.Getenv("VMNETWORK"), "The VMware vSwitch the VM will use")
-	newVM.vSphereHost = flags.String("hostname", os.Getenv("VMHOST"), "The Server that will run the VM")
+	newVM.vCenterURL = flags.String("url", os.Getenv("VCURL"), "URL of VMware vCenter in the format of https://username:password@VCaddress/sdk")
+	newVM.dsName = flags.String("datastore", os.Getenv("VCDATASTORE"), "The name of the DataStore to host the VM")
+	newVM.networkName = flags.String("network", os.Getenv("VCNETWORK"), "The network label the VM will use")
+	newVM.vSphereHost = flags.String("hostname", os.Getenv("VCHOST"), "The server that will run the VM")
```

Comment:
The VC**** environment variables are there to make it a bit more clearer when repeatedly deploying VMs. The credentials and labels for vCenter internals will stay the same, so we can store them elsewhere (env vars in this case), the rest of the flags we can pass as usual as there is some need for uniqueness. Also @rneugeba, suggested some format for the env variables so I opted for VC<resource type>.